src/redis_db.h

```diff
@@ -31,7 +31,10 @@ class Database {
                        std::vector<std::string> *keys);
   rocksdb::Status RandomKey(const std::string &cursor, std::string *key);
   void AppendNamespacePrefix(const Slice &user_key, std::string *output);
-  rocksdb::Status FindKeyRangeWithPrefix(const std::string &prefix, std::string *begin, std::string *end);
+  rocksdb::Status FindKeyRangeWithPrefix(const std::string &prefix,
```

Comment:
How about moving the `rocksdb::ColumnFamilyHandle` as the first param to keep the consistency with others